nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -2617,4 +2617,247 @@ public void orElseLibraryModelSupport() {
             "}")
         .doTest();
   }
+
+  @Test
```

Comment:
At some point, we should be breaking out new test methods into their own test classes, as `NullAwayTest` is enormous.  It will make running tests faster since Gradle can run separate test classes in parallel.  This is more a reminder for me and @lazaroclapp, does not need to be addressed here

Reply:
Might be worth extracting this reminder into an issue before we both forget again: #430 (that said, cleaning up that issue tracker a bit is also a task to not forget 😅 )